### ISSUE TITLE
Switch raiden heartbeat endpoint

### ISSUE THREAD

**Issue by kilrau** [ExchangeUnion/xud#1099]
Switch raiden heartbeat endpoint (e.g. `getinfo`) to `/api/v1/address`, at the moment it's `/api/v1/channels`, which makes the log a big hard to read and also /channels is blocked when a channel is opened and throws a lot of errors.

**Comment by sangaman** [comment]
We already are using the `/api/v1/address` endpoint to verify whether raiden is online and reachable, what exactly are you seeing in the logs?

**Comment by sangaman** [comment]
Actually I think I know what's happening here, we periodically check our available balance when balance checks are enabled to know the max order size we will accept. Does the blocked call cause problems? It's not a big problem if we have to wait for a response for the `/channels` call within xud, but if it's causes problems in raiden we can disable the scheduled balance checks for the time being.

**Comment by kilrau** [comment]
Understood, it doesn't cause problems, just shows loads of errors in the xud log, which is confusing since one gets the impression the raiden client is down. But thanks for confirming that we are already using `/address` as heartbeat (@offerm ), should have checked that myself!

In short: I wouldn't change xud behavior for now, since this would be temporary code again and I discussed with @rakanalh yesterday that raiden still should handle other calls to the `/channels` endpoint while a channel is being opened. Closing here.
